src/lib/base-oracle/HashConsensus.sol

```diff
@@ -87,6 +24,7 @@ interface IReportAsyncProcessor {
 /// given that oracle reports sometimes have to contain diffs instead of the full state which
 /// might be impractical or even impossible to transmit and process.
 ///
+// solhint-disable ordering
```

Comment:
Should we also fix "I8: HashConsensus should inherit from IConsensusContract" here?

Reply:
added

Reply:
is it added?